src/components/Tabs/TabList.tsx

```diff
@@ -14,12 +14,15 @@ export const TabList = React.forwardRef<HTMLDivElement, TabListProps>(function T
   return (
     // @ts-ignore Typing is wrong on @reach-ui, allowing only div | undefined for `as`
     <ReachTabList as={Flex} ref={ref} flexWrap="wrap" {...rest}>
-      {React.Children.map(children, (child, index) =>
-        React.cloneElement(child as React.ReactElement, {
-          isSelected: selectedIndex === index,
-          isFocused: focusedIndex === index,
-        })
-      )}
+      {React.Children.map(children, (child, index) => {
+        if (child) {
+          return React.cloneElement(child as React.ReactElement, {
+            isSelected: selectedIndex === index,
+            isFocused: focusedIndex === index,
+          });
+        }
+        return null;
+      })}
```

Comment:
I think `React.Children.toArray` filters out `null` values, so we could do `React.Children.toArray(children).map(....)`.

Can we try this?

Reply:
Yeap that seemed to also do the trick, have a look!